powersimdata/data_access/data_access.py

```diff
@@ -141,13 +105,12 @@ def execute_command_async(self, command):
         raise NotImplementedError
 
     def checksum(self, relative_path):
-        """Return the checksum of the file path, and write the content if the
-        server is remote
+        """Return the checksum of the file path
 
         :param str relative_path: path relative to root
         :return: (*str*) -- the checksum of the file
         """
-        raise NotImplementedError
+        return "dummy_value"
```

Comment:
The reason that we would like to return something here now is solely for simplification purpose for `LocalDataAccess` to inherit, right?

Reply:
Pretty much, yeah. I suppose it kinda makes sense to have as the default, for `DataAccess` subclasses that don't need to validate checksums. May move this around in the future though.